docs/source/DEVELOPER.rst

```diff
@@ -144,3 +144,29 @@ The Deepcell Kiosk uses Google Kubernetes Engine to requisition resources on Goo
 5. any Persistent Disks associated with your cluster
 
 While we hope this list is comprehensive, there could be some lingering resources used by Google Cloud and not deleted automatically that we're not aware of.
+
+Benchmarking Figures from DeepCell Kiosk Paper
+---------------------------------------------------------
+
+The DeepCell Kiosk paper (Bannon et al., 2020) presents cost and runtime benchmarks for running a generic segmentation pipeline on datasets of given sizes inside the DeepCell Kiosk with different sets of constraints (Fig. 1b).
+
+Here, we provide any interested parties with a way to recreate these figures.
+
+Recreating Figures
+^^^^^^^^^^^^^^^^^^
+
+1. First, checkout the `benchmarks` branch of the `kiosk-console` repository. This branch is almost identical to the version 1.2.0 release of the `kiosk-console` repo, except for minor configuration differences, which the user can view by executing `git diff master..benchmarks` from a command prompt inside a local copy of the repository. There are three very important variable settings worth noting, each with a corresponding setup action for the user to perform before benchmarking:
+    - In the `Dockerfile`, the `CLOUDSDK_BUCKET` environmental variable is set to `kiosk-benchmarking`, which is the Google Cloud bucket that the Van Valen Lab uses to store its benchmarking models. To recreate our benchmarking data, please change this value to the name of a bucket that you control.
```

Comment:
I'm a bit lost on the direction for both this point and the following. Is there an easier way to do/explain this?

Reply:
Yeah they do not need to change the environmental variable, they are prompted to enter their own bucket during the kiosk configuration.

Reply:
Fixed.